src/main/java/hw10/configuration.txt

```diff
@@ -0,0 +1,4 @@
+FILE- C:/hillel/hillel_homework/src/main/java/hw10/application.log
```

Comment:
Can you please replace these keys (FILE, LEVEL, etc.) with lowercase words, for example, `file, level, size, format`? Also, do not use `-` as a separator, use `=` instead.